src/controllers/controller.bar.js

```diff
@@ -383,11 +388,12 @@ module.exports = function(Chart) {
 			var meta = me.getMeta();
 			var scale = me.getValueScale();
 			var datasets = chart.data.datasets;
-			var value = scale.getRightValue(datasets[datasetIndex].data[index]);
+			var value = scale._parseValue(datasets[datasetIndex].data[index]);
 			var stacked = scale.options.stacked;
 			var stack = meta.stack;
-			var start = 0;
-			var i, imeta, ivalue, base, head, size;
+			var i, imeta, ivalue, base, head, size, yStackValue;
+			var value.start = value.max >= 0 && value.min >= 0 ? value.min : value.max;
```

Comment:
Don't override `start` here or on line 410. The values set by `_parseValue` are used in the labels and are expected to be what the user passed in. Create a new variable or add to some other existing variable, but please don't modify what is returned by `_parseValue`